static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -0,0 +1,52 @@
+import {t} from 'app/locale';
+
+export type NotificationSettingField = {
```

Comment:
should this be in our `types/index.tsx` file?

Reply:
@mgaeta bumping this

Reply:
I copied this type from `notifications/fields.tsx` so it makes sense to also keep `NotificationSettingField` in the file.